src/Sulu/Bundle/MediaBundle/Controller/AbstractMediaController.php

```diff
@@ -54,14 +54,10 @@ protected function getTitleFromUpload($request)
 
             return \implode('.', \explode('.', $uploadedFile->getClientOriginalName(), -1));
         }
+        return null;
```

Comment:
Not a bc break. No return at end of function, it will return null. So this is making implicit behavior now explicit.